nuxeo-opencmis-impl/src/main/java/org/nuxeo/ecm/core/opencmis/impl/server/CMISQLtoNXQL.java

```diff
@@ -301,6 +303,12 @@ public IterableQueryResult getIterableQueryResult(IterableQueryResult it, NuxeoC
         return new NXQLtoCMISIterableQueryResult(it, realColumns, virtualColumns, service);
     }
 
+    public PartialList<Map<String, Serializable>> getPartialList(PartialList<Map<String, Serializable>> pl,
```

Comment:
It's just to follow naming we already have: `IterableQueryResult getIterableQueryResult(IterableQueryResult it, NuxeoCmisService service)`.
I prefer to follow in place naming as we didn't deprecate the other method, wdyt ?